### PR TITLE
Update foreman_ansible to 1.4.5 [rpm]

### PR DESCRIPTION
**Pull request by dLobatog**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [x] 1.15
* [x] 1.14
* [x] 1.13

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: rubygem-foreman_ansible/rubygem-foreman_ansible.spec [hunk 1]
```diff
@@ -19,7 +19,7 @@ Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}rubygems
 Requires: %{?scl_prefix}rubygem(deface) < 2.0
 Requires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.8.1
```
Comment (domcleal) [inline rubygem-foreman_ansible/rubygem-foreman_ansible.spec]: Downgraded to `0.8` (https://github.com/theforeman/foreman_ansible/blob/1.4.4/foreman_ansible.gemspec, https://github.com/theforeman/foreman_ansible/blob/1.4.5/foreman_ansible.gemspec)

File: rubygem-foreman_ansible/rubygem-foreman_ansible.spec [hunk 2]
```diff
@@ -19,7 +19,7 @@ Requires: %{?scl_prefix_ruby}ruby(release)
...
-Requires: %{?scl_prefix}rubygem(foreman-tasks) < 0.9.0
+Requires: %{?scl_prefix}rubygem(foreman-tasks) < 1.0.0
 Requires: %{?scl_prefix}rubygem(dynflow) >= 0.8.14
 Requires: %{?scl_prefix}rubygem(dynflow) < 0.9.0
 Requires: %{?scl_prefix}rubygem(foreman_ansible_core) >= 1.1.0
```
Comment (domcleal) [inline rubygem-foreman_ansible/rubygem-foreman_ansible.spec]: 1.0

File: rubygem-foreman_ansible/rubygem-foreman_ansible.spec [hunk 3]
```diff
@@ -29,7 +29,7 @@ BuildRequires: %{?scl_prefix_ruby}rubygems-devel
 BuildRequires: %{?scl_prefix_ruby}rubygems
 BuildRequires: %{?scl_prefix}rubygem(deface) < 2.0
 BuildRequires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.8.1
-BuildRequires: %{?scl_prefix}rubygem(foreman-tasks) < 0.9.0
+BuildRequires: %{?scl_prefix}rubygem(foreman-tasks) < 1.0.0
 BuildRequires: %{?scl_prefix}rubygem(dynflow) >= 0.8.14
 BuildRequires: %{?scl_prefix}rubygem(dynflow) < 0.9.0
 BuildRequires: %{?scl_prefix}rubygem(foreman_ansible_core) >= 1.1.0
```
Comment (domcleal) [inline rubygem-foreman_ansible/rubygem-foreman_ansible.spec]: 1.0

File: rubygem-foreman_ansible/rubygem-foreman_ansible.spec [hunk 4]
```diff
@@ -29,7 +29,7 @@ BuildRequires: %{?scl_prefix_ruby}rubygems-devel
 BuildRequires: %{?scl_prefix_ruby}rubygems
 BuildRequires: %{?scl_prefix}rubygem(deface) < 2.0
 BuildRequires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.8.1
```
Comment (domcleal) [inline rubygem-foreman_ansible/rubygem-foreman_ansible.spec]: 0.8

**Comment by mmoll** [comment]
@dLobatog could you please update this PR, as #1579 is blocked on it.

**Comment by dLobatog** [comment]
@mmoll @domcleal Updated - I was not aware this was blocking foreman-tasks

**Comment by domcleal** [comment]
Merged and building, thanks @dLobatog.

**Pull request merged**